src/libxtp/ERIs.cc

```diff
@@ -192,7 +192,116 @@ namespace votca {
         }
         
         
-        
+		
+		
+		void ERIs::CalculateERIs_4c_direct(const AOBasis& dftbasis, const ub::matrix<double> &DMAT)
+		{
+			cout << endl;
+			cout << "ERIS.cc ERIs::CalculateERIs_4c_direct" << endl;
+			
+			// Initialize ERIs matrix
+			_ERIs = ub::zero_matrix<double>(DMAT.size1(), DMAT.size2());
+			// Get number of shells
+			int numShells = dftbasis.getNumofShells();
+			
+			// Test
+			printf("size(DMAT) = [%d, %d]\n", int(DMAT.size1()), int(DMAT.size2()));
+			printf("dftbasisSize = %d\n", dftbasis.AOBasisSize());
```

Comment:
Any reason for using printf(C) vs cout(C++) here?

Reply:
Thanks for the comment, Joshua!

I will work on the formatting. Also, I'm used to methods similar to printf from other languages. Guess it would make more sense to use cout in this case!